packages/rest/src/validation/request-body.validator.ts

```diff
@@ -92,15 +88,16 @@ const compiledSchemaCache = new WeakMap();
 function validateValueAgainstSchema(
   // tslint:disable-next-line:no-any
   body: any,
-  schema: SchemaObject,
+  schema: SchemaObject | ReferenceObject,
```

Comment:
Hmm, any reason remove the `ReferenceObject` here?

Reply:
It's adding `ReferenceObject` back.